ui/app/src/components/Controls/SingleFileUpload.tsx

```diff
@@ -25,6 +25,8 @@ import { defineMessages, useIntl } from 'react-intl';
 import '@uppy/core/src/style.scss';
 import '@uppy/progress-bar/src/style.scss';
 import '@uppy/file-input/src/style.scss';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import createStyles from '@material-ui/core/styles/createStyles';
```

Comment:
Past the third level imports are considered private, import { makeStyles, createStyles } from @material-ui/core/styles